developers/contributor-guide/current/weaviate-contributors-guide/commit-guidelines.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: layout-documentation
+solution: contributor-guide
+sub-menu: Weaviate Contributor's Guide
+title: Commit Guidelines 
```

Comment:
I'm not sure this belongs in the contributor guide

Reply:
Is there any other place where we could have this?
I believe we should provide guidelines on how to perform commits according to our standards.

Reply:
done